api/services/administration/general/paths/getPolicyAction.yaml

```diff
@@ -0,0 +1,18 @@
+get:
+  summary: "Get Access Policy Actions"
+  description: Get actions for access policies.
+  operationId: "OrganizationsAndAccountsGetAccessPolicyAction"
+  tags:
+    - "Organizations"
+  parameters:
```

Comment:
lets move all parameters to: `api/services/administration/general/parameters` as it is only relevant to policy action

Reply:
done